src/index.js

```diff
@@ -72,6 +84,22 @@ var Index = new Lang.Class({
             doc.add_numeric_value(SEQUENCE_NUMBER_VALUE_NO, metadata['sequenceNumber']);
         }
 
+        const indexingDateString = _indexDateForMetadata(metadata);
+        if (indexingDateString) {
+            /* Parse the date using the native JS Date library, since
+             * GLib.Date.set_parse isn't able to handle JS-style dates. */
+            try {
+                const parsedIndexingDate = new Date(indexingDateString)
```

Comment:
Missing semicolon

Reply:
The woes of switching between Python and JS ;-)